src/main/java/org/fcrepo/storage/ocfl/validation/ObjectValidator.java

```diff
@@ -55,13 +55,13 @@
 
     private final MutableOcflRepository ocflRepo;
     private final ObjectReader headerReader;
-    private final HeadersValidator headersValidator;
+    private final DefaultHeadersValidator headersValidator;
 
```

Comment:
Are we requiring javadocs on public constructors? I'm surprised the checkstyle passed...

Reply:
As noted by @bbpennel, the javadoc checks do not currently seem to be working for whatever reason. That said, while I do think javadoc is important in most cases, I also think that it can be pointless, clutter. For example, I could add javadoc to this class constructor like:

```
/**
     * Creates a new object validator
     * 
     * @param ocflRepo the ocfl repository
     * @param headerReader the resourece header deserializer
     */
```

But, to me, that is no more obvious than if the constructor did not have javadoc, and adds 6 lines of clutter.

Reply:
That is a valid point.